File: app/src/main/java/com/chelomin/wallaby/model/ProductListDto.java
package com.chelomin.wallaby.model;

import java.util.List;

/**
 * Created by huge on 12/8/17.
 */

public class ProductListDto {
    public List<ProductDto> products;
    public Integer totalProducts;
    public Integer pageNumber;
    public Integer pageSize;
    public Integer status;
    public String kind;
    public String etag;
}

File: app/src/main/java/com/chelomin/wallaby/model/ProductListDto.kt
package com.chelomin.wallaby.model

/**
 * Created by huge on 12/8/17.
 */

class ProductListDto {
    var products: List<ProductDto>? = null
    var totalProducts: Int? = null
    var pageNumber: Int? = null
    var pageSize: Int? = null
    var status: Int? = null
    var kind: String? = null
    var etag: String? = null
}
